docs/server/source/http-client-server-api.rst

```diff
@@ -157,21 +157,29 @@ Transaction Outputs
 -------------------
 
 The ``/api/v1/outputs`` endpoint returns transactions outputs filtered by a
-given public key, and optionally filtered to only include outputs that have
-not already been spent.
+given public key, and optionally filtered to only include either spent or
+unspent outputs.
 
 
-.. http:get:: /api/v1/outputs?public_key={public_key}
+.. http:get:: /api/v1/outputs
 
-   Get transaction outputs by public key. The `public_key` parameter must be
+   Get transaction outputs by public key. The ``public_key`` parameter must be
    a base58 encoded ed25519 public key associated with transaction output
    ownership.
 
-   Returns a list of links to transaction outputs.
+   Returns a list of transaction outputs.
+
+   :param public_key: Base58 encoded public key associated with output
+                      ownership. This parameter is mandatory and without it
+                      the endpoint will return a ``400`` response code.
+   :param spent: Boolean value ("true" or "false") indicating if the result set
+                 should include only spent or only unspent outputs. If not
+                 specified the result includes all the outputs (both spent
+                 and unspent).
```

Comment:
I guess you mean, "...all the outputs (both spent and unspent) associated with the `public_key`."